docs/network/fees.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Fee structure
+---
+
+Etherlink transactions include two fees:
+
+- The _gas fee_ changes depending on Etherlink throughput over time.
+As the demand for Etherlink transactions increases, the gas fee rises.
+- The _storage fee_ is proportional to the amount of data that Etherlink stores on Tezos layer 1.
```

Comment:
storage fee might be slightly misleading?  perhaps `inclusion fee` would be a better term - it's the cost of including the tx in an L1 block, as opposed to storing it permanently in the L1 context/storage

Reply:
Changed to "inclusion fee."